docs/rules/README.md

```diff
@@ -139,9 +138,9 @@ These rules are specific to JavaScript running on Node.js or using CommonJS in t
 * [no-mixed-requires](no-mixed-requires.md) - disallow mixing regular variable and require declarations
 * [no-new-require](no-new-require.md) - disallow use of `new` operator with the `require` function
 * [no-path-concat](no-path-concat.md) - disallow string concatenation with `__dirname` and `__filename`
+* [no-process-env](no-process-env.md) - disallow use of `process.env`
 * [no-process-exit](no-process-exit.md) - disallow `process.exit()`
-* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified Node.js imports
-* [no-restricted-modules](no-restricted-modules.md) - restrict usage of specified Node.js modules
+* [no-restricted-modules](no-restricted-modules.md) - restrict usage of specified modules when loaded by `require` function [see https://nodejs.org/api/globals.html#globals_require and https://nodejs.org/api/modules.html#modules_addenda_package_manager_tips]
```

Comment:
@brettz9 uh oh 😱 my bad to include the cross-references in square brackets. which were for your benefit in a way that looked as if they belonged in the list